src/main/java/edu/wpi/first/outlineviewer/view/dialog/PreferencesDialog.java

```diff
@@ -9,7 +10,9 @@
 /**
  * A dialog for changing the app preferences.
  */
-public class PreferencesDialog extends Dialog<ButtonType> {
```

Comment:
This was intentional to allow for finer grained results instead of just accept/cancel.